airflow_clickhouse_plugin/sensors/clickhouse_sql_sensor.py

```diff
@@ -0,0 +1,61 @@
+from airflow.exceptions import AirflowException
+from airflow.sensors.sql_sensor import SqlSensor
+from airflow_clickhouse_plugin.hooks import clickhouse_hook
+
+
+class ClickHouseSqlSensor(SqlSensor):
+    def __init__(
+            self,
+            sql: str = None,
+            clickhouse_conn_id: str = 'clickhouse_default',
+            parameters=None,
+            database: str = None,
+            success=None,
+            failure=None,
+            fail_on_empty=False,
+            *args,
+            **kwargs,
+    ):
+        super(ClickHouseSqlSensor, self).__init__(
+            conn_id=clickhouse_conn_id,
+            sql=sql,
+            parameters=parameters,
+            success=success,
+            failure=failure,
+            fail_on_empty=fail_on_empty,
+            *args,
+            **kwargs,
+        )
+        self._database = database
+
+    def poke(self, context):
+        """
+        Monkey patch the original poke to use ClickHouseHook
+        https://github.com/apache/airflow/blob/73bf718358c01cc41e5f23b914a8824a2665a28c/airflow/sensors/sql_sensor.py
+        """
+
+        hook = clickhouse_hook.ClickHouseHook(
+            clickhouse_conn_id=self.conn_id, database=self._database,
+        )
+
+        self.log.info("Poking: %s (with parameters %s)", self.sql, self.parameters)
```

Comment:
I will be glad if you use single-quoted string even in the copy-pasted code to keep codebase within a consistent code style. Please notice there are some other double-quoted strings to be fixes. Thank you in advance.